### ISSUE TITLE
Java >= 9 compatibility

### ISSUE THREAD

**Issue by harti2006** [Rothamsted/neo4j-server-maven-plugin#4]
Currently, when executing the plugin in an JDK 9 environment, it fails to unpack the neo4j archive properly:

```
[INFO] Extracting /var/folders/q2/5yhk0jjx5pnd45xtjjbs90cc0000gn/T/neo4j-server-maven-plugin/downloads/server/3.3.1/neo4j-server-unix.tar.gz
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.rauschig.jarchivelib.FileModeMapper$FileSystemPreferencesReflectionChmodCommand (file:~/.m2/repository/org/rauschig/jarchivelib/0.7.1/jarchivelib-0.7.1.jar) to method java.util.prefs.FileSystemPreferences.chmod(java.lang.String,int)
WARNING: Please consider reporting this to the maintainers of org.rauschig.jarchivelib.FileModeMapper$FileSystemPreferencesReflectionChmodCommand
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

# Followed by warnings for each file:
März 23, 2018 9:20:21 VORM. org.rauschig.jarchivelib.FileModeMapper$UnixPermissionMapper chmod
WARNUNG: Could not set file permissions of ~/projects/neo4j-server-maven-plugin/integration-tests/target/neo4j-server/neo4j-community-3.3.1. Exception was: null```

**Comment by AdmiralGaust** [comment]
I am having similar issues with neo4j server. I am using neo4j to work with BloodHound. The server is running well but when I try to upload files to BloodHound it says unrecognized file format. When I see the log file for neo4j it is having following contents

```nohup: ignoring input
2018-09-20 10:21:23.891+0000 WARN  You are using an unsupported version of the Java runtime. Please use Oracle(R) Java(TM) Runtime Environment 8, OpenJDK(TM) 8 or IBM J9.
2018-09-20 10:21:23.916+0000 INFO  ======== Neo4j 3.4.7 ========
2018-09-20 10:21:23.969+0000 INFO  Starting...
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.neo4j.unsafe.impl.internal.dragons.UnsafeUtil (file:/home/shiva/BloodHound/neo4j-community-3.4.7/lib/neo4j-unsafe-3.4.7.jar) to constructor java.lang.String(char[],boolean)
WARNING: Please consider reporting this to the maintainers of org.neo4j.unsafe.impl.internal.dragons.UnsafeUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release```

**Comment by harti2006** [comment]
fixed with #10
